.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.idea/
+package-lock.json
```

Comment:
`package-lock.json` should not be ignored.
https://nodejs.dev/learn/the-package-lock-json-file